practice/practice-02/src/includes/mixins.pug

```diff
@@ -0,0 +1,25 @@
+- var values = [1, 2, 3];
+
+mixin cardFoodItem(itemFood, price, ratePoint, foodName, foodDescription)
```

Comment:
Could we use object here? It's hard to remember the order of params

Reply:
Btw, itemFood is not a good name

Reply:
Fixed.